packages/core/lib/dlc/finance/Builder.ts

```diff
@@ -168,7 +168,7 @@ export const buildOrderOffer = (
   orderOffer.offerCollateralSatoshis = totalCollateral - BigInt(premium);
   orderOffer.feeRatePerVb = BigInt(feePerByte);
 
-  orderOffer.cetLocktime = expiration;
+  orderOffer.cetLocktime = Math.floor(new Date().getTime() / 1000); // set to current time
```

Comment:
According to Chris Belcher on bitcoin mailing list 

```
When wallets create transactions spending UTXOs protected by BIP341
taproot, they should set either an nLockTime value or nSequence values
to discourage fee sniping, by allowing the transaction to only be mined
in the next block after the tip, not the current block. This BIP
suggests 50% probability for using nLockTime and 50% for nSequence. If
nSequence is set it should apply only to the first input of the
transaction, if it has multiple inputs.
```

https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-June/019048.html

We should probably add another parameter to `buildOrderOffer` that includes block height or cetLocktime so next block can be passed in

https://github.com/AtomicFinance/node-dlc/blob/master/packages%2Fcore%2Flib%2Fdlc%2Ffinance%2FBuilder.ts#L81-L91